Gaussian Mixture Model/README.md

```diff
@@ -0,0 +1,202 @@
+
+\
+<space>
+The result of this operation would be high when the point is assigned to the right cluster and lower otherwise.
+
+**Step 3 (Maximization step):**
+
+Here we go back and update the &Pi;, &micro; and &sum; values. These are updated in the following manner:
+
+1. The new density is defined by the ratio of the number of points in the cluster and the total number of points:  
+    \
+   <space>
+   ![Pi update](https://cdn.analyticsvidhya.com/wp-content/uploads/2019/10/mean-formula-gmm.png)
+
+2. The mean and the covariance matrix are updated based on the values assigned to the distribution, in proportion with the probability values for the data point. Hence, a data point that has a higher probability of being a part of that distribution will contribute a larger portion:
+
+   \
+   <space>
+   ![Mu and Sigma Update](https://cdn.analyticsvidhya.com/wp-content/uploads/2019/10/formulas-gmm-e1571990956378-300x128.png)
+
+\
+ <space>
+
+**Step 4 :**
+
+Now based on the updated values generated from this step , we calculate the probabilities for each data points and update the values iteratively. This is done in order to maximize the [log-likelihood function](https://en.wikipedia.org/wiki/Likelihood_function#:~:text=Log%2Dlikelihood%20function%20is%20a,to%20maximizing%20the%20log%2Dlikelihood.).
+
+In other words, _k-means only considers the mean to update the centroid while GMM takes into account the mean as well as the variance of the data._
+
+> ![example of EM](https://miro.medium.com/max/450/0*kE-YHM1yJfxnbCLt.gif) \
+>  <space> [Expectation Maximization for Old Faithful Eruption Data](https://en.wikipedia.org/wiki/Expectation%E2%80%93maximization_algorithm)
+
+\
+<space>
+
+## Expectation Maximization in General
+
+1. We have some data X of whatever form.
+2. We posit there is also unobserved (latent) data Δ, again of whatever form.
+3. We have a model with parameters &theta;.
+4. We have the ability to compute the log-likelihood ℓ(θ; X, Δ). Specifically, the log of the probability of observing our data and specified assignments of the latent variables given the parameters.
+5. We also have the ability to use the model to compute the conditional distribution Δ|X given a set of parameters. We will denote this P(Δ|X; θ).
+6. Consequently we can compute the log-likelihood ℓ(θ; X). This is the log of the probability of observing our data given the parameters (without assuming an assignment for the latent variables).
+
+_More about log-likelihood [here](https://en.wikipedia.org/wiki/Likelihood_function#:~:text=Log%2Dlikelihood%20function%20is%20a,to%20maximizing%20the%20log%2Dlikelihood.)_
+
+\
+<space>
+**References :**
+
+1. [Elements of Statistical Learning](https://web.stanford.edu/~hastie/ElemStatLearn/)
+
+2. [Scikit Learn](hhttp://scikit-learn.org/stable/modules/mixture.html#:~:text=A%20Gaussian%20mixture%20model%20is,Gaussian%20distributions%20with%20unknown%20parameters.)
```

Comment:
Link not working. 
Please make the necessary changes.
![image](https://user-images.githubusercontent.com/47635806/116468153-2389fc00-a88e-11eb-8368-04ca52ba895d.png)